handbook/engineering/extensibility/index.md

```diff
@@ -0,0 +1,66 @@
+# Extensibility team
+
+The extensibility team owns our code host and third-party integrations (including our browser extension) and our [Sourcegraph extensions](https://docs.sourcegraph.com/extensions).
+
+## Contact
+
+- [#extensibility-chat](https://app.slack.com/client/T02FSM7DL/C01LZKLRF0C) channel or @extensibility on Slack
+- On GitHub, mention or assign issues to the [@sourcegraph/extensibility](https://github.com/orgs/sourcegraph/teams/extensibility) team.
+
+## Ownership
+
+This team is responsible for:
+
+1. Bringing the value of Sourcegraph to other developer tools
```

Comment:
```suggestion
1. Bringing the value of Sourcegraph to everywhere you work with code
```
What about if we appear on Stripe API Docs or Stack Overflow? Are docsites "tools"? (Tools about the tools they document?) Is Slack -- for a Slack integration -- a developer tool? I'd prefer to make this much broader in order to encourage people to think of new uses/not limit to traditional developer tools as we see fit! A less-exciting but less-vague version of this is "to other tools and websites"

Reply:
Good point, I'll limit our use of "developer tools" in the team page, since it can be seen as both vague and narrow